source/build/Android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
+        classpath 'com.android.tools.build:gradle:3.5.2'
```

Comment:
Somehow my comment didn't get posted? 

You should check out this Gradle doc page: https://docs.gradle.org/current/userguide/single_versions.html 

There's suggestions for larger projects to use "upgrade-able" ranges for versions numbers OR we could use constraints and wouldn't need to explicitly label the version number itself.

Reply:
My previous comments weren't showing up on the review. I'm closing this one as it's an accidental duplicate.